packages/utils/telemetry-utils/src/config.ts

```diff
@@ -7,17 +7,24 @@ import { Lazy } from "@fluidframework/core-utils";
 import { createChildLogger, tagCodeArtifacts } from "./logger";
 import { ITelemetryLoggerExt } from "./telemetryTypes";
 
+/**
+ * @public
+ */
 export type ConfigTypes = string | number | boolean | number[] | string[] | boolean[] | undefined;
 
 /**
- * Base interface for providing configurations to enable/disable/control features
+ * Base interface for providing configurations to enable/disable/control features.
+ *
+ * @public
  */
 export interface IConfigProviderBase {
 	getRawConfig(name: string): ConfigTypes;
 }
 
 /**
- * Explicitly typed interface for reading configurations
+ * Explicitly typed interface for reading configurations.
+ *
+ * @public
```

Comment:
Maybe @internal

Reply:
And maybe we can move that stuff to `telemetryTypes` (rename that file).  I'm wondering if we can manage to only export types from this package, and only very few.

Reply:
+1 on `IConfigProvider` in particular being internal. Confirmed no uses in bohemia nor loopsdk.